settings.gradle

```diff
@@ -113,8 +113,12 @@ project(":code/red-black-tree").name = "red-black-tree-Model"
 // Попков Кирилл
 // Разин Вячеслав
 // Сизова Ксения
```

Comment:
One comment: Pls, rename to your project name: for example - Binary Search

Reply:
Done!